firebase-installations/customer-lint-checks/src/main/java/com/google/firebase/installations/lint/IncompatibleIidVersionDetector.kt

```diff
@@ -14,22 +14,15 @@
 
 package com.google.firebase.installations.lint
 
-import com.android.tools.lint.detector.api.Detector
-
 import com.android.builder.model.MavenCoordinates
-import com.android.tools.lint.detector.api.Category
-import com.android.tools.lint.detector.api.Context
-import com.android.tools.lint.detector.api.Implementation
-import com.android.tools.lint.detector.api.Issue
-import com.android.tools.lint.detector.api.Location
-import com.android.tools.lint.detector.api.Scope
-import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.*
```

Comment:
This is against Java readability rules in Google I think. Not sure how this is handled here?